dockers/docker-platform-monitor/base_image_files/cmd_wrapper

```diff
@@ -7,4 +7,4 @@ if [ -t 1 ] ; then
     DOCKER_EXEC_FLAGS+="t"
 fi
 
-docker exec -$DOCKER_EXEC_FLAGS pmon sensors "$@"
+docker exec -$DOCKER_EXEC_FLAGS pmon $(basename $0) "$@"
```

Comment:
@andriymoroz-mlnx: Just wondering if we need this script at all anymore, especially because now you need to pass the binary name to the file. The point of these wrapper scripts is to allow users to run executables which reside inside Docker containers as if they are in the base image. If you think there will still be a need for this, then we can leave it, but I think you might want to consider renaming it `pmon_cmd_wrapper`.